src/validator.ts

```diff
@@ -30,7 +30,9 @@ export class Validator {
     } else {
       validate = ajv.compile(schema);
     }
-    if (validate === undefined) return null;
+    if (validate === undefined) {
+      throw new Error(`Invalid schema: '${schema}'`);
```

Comment:
That doesn't seem very helpful if the schema is not a string.

Reply:
Also no need to use a block.

Reply:
Looks like `ajv` throws for actual schema errors, so the message is probably fine.